bin/nrdebug

```diff
@@ -155,7 +155,7 @@ class RubyProcess
   def gdb_script_body(backtrace_file)
     backtrace_gathering_code = 'Thread.list.each { |t| bt = t.backtrace rescue nil; ' \
       'puts \"#{t.inspect}\n#{bt && bt.join(\"\n\")}\n\n\" }'
-    <<-END
+    <<~END
```

Comment:
This change from `<<-END` to `<<~END` changes the output of the `puts` command. Previously, all of the leading whitespace was printed. Now it gets chopped off:

```shell
        attach 123
        t a a bt
```

vs

```
attach 123
t a a bt
```

Reply:
Are we wanting the whitespace to remain?

Reply:
This was the first modified heredoc I had seen when looking at the PR and I commented before realizing that elsewhere you had done a lot of reformatting on a case by case basis. If this is outputting the text in a format you approve of, that's certainly good enough for me.

Reply:
Yes I'm happy with it!